src/website/app/demos/Button/examples/disabled.js

```diff
@@ -1,37 +0,0 @@
-/**
```

Comment:
I thought about that too, and the reason I ended up combining them is that `disabled` is not a prop that Mineral UI invented, so I figured it didn't need to be called out specifically. Especially since our disabled styles are really just an extension of the default disabled browser styles.

Reply:
Fair enough. I guess Popover and Dropdown will be exceptions, since they require additional instruction.